docs/aggregate.md

```diff
@@ -22,6 +22,10 @@ ds2 = ds.aggregate(group_by=['FIPS'], aggregations={
           'COL_2': ['SUM', 'AVG']
       })
 ds2.preview()
+
+ds3 = ds.aggregate(aggregations={'COL_1': ['SUM']})
+
+ds4 = ds.aggregate(aggregations={'DISTINCT COL_1': ['SUM']})
```

Comment:
i think inputting DISTINCT as part of the column specification might be confusing, and also break the UI input, vs treating it as a new agg method

Reply:
you can do  `select sum(distinct column)` and same with all the other aggregations.  If we relegate it to the aggregation functions, I think that means we have to disallow that use case. Another option is to make the aggregation object a little more complicated and do something like 

```
aggregations={
  'Col1' : {
    aggregations: ['SUM', 'AVG', ...],
    distinct: true
    }
}
```

But that would seem to imply that you want the distinct aggregation of all the aggregations you select.  Is that preferable?  It would also make it legal to add `Col1` twice, once as a distinct set of aggregations and another as not. an if you just want to `SELECT DISTINCT Col1 FROM wherever` you could just pass in `distinct: true` with no aggregations.

But that's getting way more complicated than usable.  Thoughts?